### PR TITLE
fix vulnerabilities (publish-please, ws)

### PR DESCRIPTION
**Pull request by Farfurix**
Before:
```
found 20 vulnerabilities (6 low, 11 moderate, 3 high)
```

Current:
```
found 13 vulnerabilities (5 low, 7 moderate, 1 high)
```

### Changes
1. Update `publish-please`.
2. Update `ws`.

### `npm install --save-dev publish-please@3.0.1`
**Workaround:** update `npm` version in Travis `nvm install 6` case.

### `npm install --save-dev ws@5.2.0`
https://github.com/websockets/ws/releases/tag/4.0.0:

> The non-standard `protocolVersion` and `bytesReceived` attributes have been
> removed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ addons:
    - dbus
 
 before_install:
+ # Update npm version (GH-1644)
```
Comment (miherlosev) [inline .travis.yml]: Remove this comment and add link to the issue from publish-please repository

File: .travis.yml [hunk 2]
```diff
@@ -8,6 +8,13 @@ addons:
    - dbus
 
 before_install:
+ # https://github.com/inikulin/publish-please/issues/66
+ - |
+   npm_version=$(npm -v);
```
Comment (miherlosev) [inline .travis.yml]: Simplify to `if  npm_version < 3 then update to the latest`

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Pull request merged**